src/ploigos_step_runner/step_implementers/uat/maven_selenium_cucumber.py

```diff
@@ -228,10 +232,32 @@ def _run_step(self): # pylint: disable=too-many-locals,too-many-statements
                     step_result.message = "No user acceptance tests defined" \
                         f" using maven profile ({uat_maven_profile})," \
                         " but 'fail-on-no-tests' is False."
+            else:
+                attribs = ["time", "tests", "errors", "skipped", "failures"]
+                xml_element = "testsuite"
+
+                report_results = aggregate_xml_element_attribute_values(
+                    test_results_dir,
+                    xml_element,
+                    attribs)
+
+                for attrib in attribs:
+                    if attrib in report_results:
+                        step_result.add_evidence(
+                                description="Surefire report value for " + attrib,
+                                name="uat-evidence-" + attrib,
+                                value=report_results[attrib]
+                            )
+                    else:
+                        raise ValueError("Attribute " + attrib +" not found in report")
```

Comment:
can we make this error more clear becasue it will be seen by users. something like `Error gathering evidence from surfire report, expected attribute ({attrib}) not found in report ({test_results_dir}).`

Reply:
Will do.